.github/workflows/package.yml

```diff
@@ -0,0 +1,52 @@
+name: package into static linked wheel
+
+on:
+  push:
+    tags:
+    - 'v*' #
+
+jobs:
+    package:
+        name: package up into a nice wheel
+        runs-on: ubuntu-latest
+        
+        steps:
+        - uses: actions/checkout@v2
+
+        - name: Build manylinux Python wheels
+          uses: RalfG/python-wheels-manylinux-build@v0.3.3-manylinux2014_x86_64
```

Comment:
I'm a bit worried about using 3rd party github actions. Is there anything more official?
Will this be able to access to PYPI API keys?

Reply:
I believe it wouldn't have access to secrets unless exposed through environment variables, but good point nevertheless. 

I took at look at how matplotlib builds their wheels and followed their lead.  It was  easy to swap this action out with the `cibuildwheel` package, which keeps things closer to home and seems better route overall.